### PR TITLE
Add RE tools to Testing Tools

### PR DESCRIPTION
**Pull request by joscandreu**
Added Ghidra, Binary ninja and Retdec to the list.

### CODE REVIEW

File: Document/0x08-Testing-Tools.md [hunk 1]
```diff
@@ -30,8 +30,11 @@ To perform security testing different tools are available in order to be able to
 
 ### Reverse Engineering and Static Analysis
 
+- [Binary ninja](https://binary.ninja/ "Binary ninja") - Binary ninja is a multi-platform software disassembler that can be used against that can be used against several executable file formats. It is capable of IR (intermediate representation) lifting.
```
Comment (sushi2k) [inline Document/0x08-Testing-Tools.md]: Please remove "that can be used against", as it's in there two times

File: Document/0x08-Testing-Tools.md [hunk 2]
```diff
@@ -30,8 +30,11 @@ To perform security testing different tools are available in order to be able to
 
 ### Reverse Engineering and Static Analysis
 
+- [Binary ninja](https://binary.ninja/ "Binary ninja") - Binary ninja is a multi-platform software disassembler that can be used against that can be used against several executable file formats. It is capable of IR (intermediate representation) lifting. 
+- [Ghidra](https://ghidra-sre.org/ "Ghidra") - Ghidra is an open source software reverse engineering suite of tools developed by NSA. Its main capabilities include disassembly, assembly, decompilation, graphing, and scripting.
```
Comment (sushi2k) [inline Document/0x08-Testing-Tools.md]: NSA -> the National Security Agency (NSA)

File: Document/0x08-Testing-Tools.md [hunk 3]
```diff
@@ -30,8 +30,11 @@ To perform security testing different tools are available in order to be able to
...
 - [IDA Pro](https://www.hex-rays.com/products/ida/index.shtml "IDA Pro") (Commercial Tool) - IDA is a Windows, Linux or macOS hosted multi-processor disassembler and debugger.
 - [Radare2](https://www.radare.org/r/ "Radare2") - Radare2 is a unix-like reverse engineering framework and command line tools.
+- [Retargetable decompiler](https://retdec.com/ "Retdec") - RetDec is an open-source machine-code decompiler based on LLVM. It can be used as a standalone program or as a plugin for IDA Pro or Radare2.
```
Comment (sushi2k) [inline Document/0x08-Testing-Tools.md]: open-source -> open source

**Comment by joscandreu** [comment]
I addressed the comments in the new commit.

**Comment by sushi2k** [comment]
Awesome! Thanks for adding @joscandreu

**Pull request merged**